assets/design-tokens/tailwind-colors.generated.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * ---------------------------------------------------
+ * 👷 Generated by figma2tailwind. Do not modify !
```

Comment:
why is this file committed if it's generated?

Reply:
You didn't read my PR comment? x)

Reply:
comment you mentioned: "Removes gitignore for generated files. Ignoring files is good when the files are only used in development, or are very environment-specific. But these files are simple and constant and used at runtime, so it's helpful to version-control them."

I'm not sure if I agree with that. Let's talk about it tomorrow